presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,284 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.common.type.BigintType;
+import com.facebook.presto.common.type.BooleanType;
+import com.facebook.presto.common.type.DoubleType;
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.common.type.VarcharType;
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.testing.MaterializedRow;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.testng.Assert.fail;
+
+public class ContainerQueryRunnerUtils
+{
+    private ContainerQueryRunnerUtils() {}
+
+    static Path createTpchProperties()
+            throws IOException
+    {
+        Path tempTpchProperties = Files.createTempFile("tpch", ".properties");
+        String properties = "connector.name=tpch\n" +
+                "tpch.column-naming=STANDARD\n";
+        Files.write(tempTpchProperties, properties.getBytes(), StandardOpenOption.APPEND);
+        return tempTpchProperties;
+    }
+
+    static Path createNativeWorkerConfigProperties(int coordinatorPort)
+            throws IOException
+    {
+        Path tempConfigProperties = Files.createTempFile("config", ".properties");
+        String properties = "presto.version=testversion\n" +
+                "http-server.http.port=7777\n" +
+                "discovery.uri=http://presto-coordinator:" + coordinatorPort + "\n" +
+                "system-memory-gb=2\n" +
+                "native.sidecar=false\n";
+        Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+        return tempConfigProperties;
+    }
+
+    static Path createCoordinatorConfigProperties(int port)
+            throws IOException
+    {
+        Path tempConfigProperties = Files.createTempFile("config", ".properties");
+        String properties = "coordinator=true\n" +
```

Comment:
Could you use this function https://github.com/prestodb/presto/blob/master/presto-native-execution/src/test/java/com/facebook/presto/nativeworker/NativeQueryRunnerUtils.java#L41 to retrieve properties instead of hard-coding them. Would be better to reuse.

Reply:
I have reused this in the code, can you please have a look?